app/services/bulk_tagging/search.rb

```diff
@@ -27,7 +27,8 @@ def gds_response
         document_type: document_type,
         page: page,
         q: query,
-        fields: [:content_id, :document_type, :title, :base_path]
+        fields: [:content_id, :document_type, :title, :base_path],
+        search_in: 'title,base_path,details.internal_name'
```

Comment:
We could change it in the publishing api though if we're the only ones using it. I think it's worth doing as it's a bit surprising for it not to work like all other array fields.